docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -141,12 +141,28 @@ $ cd opt/alluxio
 $ bin/alluxio runTests
 ```
 
-### Sharing ramdisk with clients
+### Read/Write to the local worker 
 
-Running the worker with `-v /mnt/ramdisk:/mnt/ramdisk` shares the ramdisk between the worker
-and host. To make this ramdisk available to clients in other containers running on the same host,
+There are two options to read/write to the local worker efficiently.
+
+#### Share ramdisk with clients
+To make this ramdisk available to clients in other containers running on the same host,
 those containers should also be run with `-v /mnt/ramdisk:/mnt/ramdisk`.
 
+#### Enable domain socket
+
+From host machine
+```bash
+$ mkdir /tmp/domain
+$ chmod a+w /tmp/domain
+$ touch /tmp/domain/domain
+$ chmod a+w /tmp/domain/domain
```

Comment:
I would suggest to use a different example, the repetition of `domain` can lead to confusion.